Source/Scene/ModelExperimental/getModelClass.js

```diff
@@ -0,0 +1,18 @@
+import ExperimentalFeatures from "../../Core/ExperimentalFeatures.js";
+import ModelExperimental from "./ModelExperimental.js";
+import Model from "../Model.js";
+/**
+ * Select either {@link Model} or {@link ModelExperimental} depending on
+ * an experimental flag. This is a cleaner way to select between them at
+ * runtime.
+ *
+ * @return {Model|ModelExperimental} The class of the Model to use.
+ *
+ * @private
+ */
+export default function getModelClass() {
```

Comment:
Was there a reason for having this as a separate file rather than as a member of `ExperimentalFeatures`? I think it's fine either way but don't remember if we discussed it much.

Reply:
it was a bit arbitrary, weren't sure whether to group it with Model where it gets used or group it with `ExperimentalFeatures`.

Though I feel `ExperimentalFeatures.getModelClass()` might look a bit odd to someone if you saw it in the code for B3DM content